source/_components/light.hue.markdown

```diff
@@ -29,12 +29,22 @@ light:
   host: DEVICE_IP_ADDRESS
 ```
 
+The light component supports multiple entries in <code>configuration.yaml</code> by appending a sequential number to the section: <code>light 2:</code>, <code>light 3:</code> etc.  When adding in a second hub, add the following lines to your `configuration.yaml`:
```

Comment:
For me, I added this example in since if a user is having trouble adding in a second physical hub, they are going to go to the Hue Hub section of the documentation.  This will solve their pain.  The broader information in the light section might be overlooked. - It was by me. 

But no offense taken if you toss this change.  I just put it out there for you guys to decide. :)